packages/office-ui-fabric-react/src/components/Button/examples/fluent/FluentTheme.ts

```diff
@@ -61,16 +61,34 @@ export const PlannerFluentTheme: IFluentThemeShape = {
     fontFace: 'Futura'
   },
   components: {
-    FluentButton: FluentButtonTheme
+    FluentButton: FluentButtonTheme,
+    FluentMenu: {
+      styles: () => ({
+        root: {
+          border: '1px solid red'
+        }
+      }),
+      variants: {
+        rounded: {
+          true: {
+            root: { borderRadius: '10px' }
```

Comment:
- FluentMenu is not deifning styles for the FluentMenuItem, it just sends props (rounded, underlined..)
- FluentMenu defines the styles for the items too, but then it is problematic if you don't want to apply some of them on one item... (`<Menu item={[{rounded: true}, {rounded: false}]} />`)